### PR TITLE
Update compare-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
Added Upgrade option to the document as this document talks about comparing groups

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2164

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/create-groups/compare-groups.md [hunk 1]
```diff
@@ -74,6 +74,12 @@ Users with permissions to the group mailbox can send as or send on behalf of the
 
 Currently it's not possible to migrate a shared mailbox to a Microsoft 365 group. Is this something you want? Let us know. **[Vote here](https://go.microsoft.com/fwlink/?linkid=871518)**.
 
+# Upgrade distribution lists to Microsoft 365 Groups in Outlook
+
+You can upgrade distribution lists to Microsoft 365 Groups with Outlook. This is a great way to give your organization's distribution lists all the features and functionality of Microsoft 365 Groups. [Why you should upgrade your distribution lists to groups in Outlook](https://support.microsoft.com/office/7fb3d880-593b-4909-aafa-950dd50ce188)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/compare-groups.md]:
```suggestion
You can upgrade distribution lists to Microsoft 365 Groups with Outlook. This is a great way to give your organization's distribution lists all the features and functionality of Microsoft 365 Groups. [Why you should upgrade your distribution lists to groups in Outlook](https://support.microsoft.com/office/7fb3d880-593b-4909-aafa-950dd50ce188).
```

File: microsoft-365/admin/create-groups/compare-groups.md [hunk 2]
```diff
@@ -74,6 +74,12 @@ Users with permissions to the group mailbox can send as or send on behalf of the
...
+
+You can upgrade DLs one at a time, or several at the same time. For more information, See [Upgrade distribution lists to Microsoft 365 Groups in Outlook](https://docs.microsoft.com/microsoft-365/admin/manage/upgrade-distribution-lists?view=o365-worldwide)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/compare-groups.md]:
```suggestion
You can upgrade distribution lists one at a time, or several at the same time. For more information, See [Upgrade distribution lists to Microsoft 365 Groups in Outlook](https://docs.microsoft.com/microsoft-365/admin/manage/upgrade-distribution-lists?view=o365-worldwide).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MikePlumleyMSFT Please merge this PR. Thanks

**Pull request merged**